src/components/Map/GoogleMapComp.tsx

```diff
@@ -1,28 +1,25 @@
-import { useEffect, useState } from 'react';
+import getCoordsFromAddress, { Coord, House } from '@/utils/services/getCoordsFromAddress';
+import { Dispatch, SetStateAction, useEffect, useState } from 'react';
 import { GoogleMap, Marker, withGoogleMap, withScriptjs } from 'react-google-maps';
-
-import getCoordsFromAddress from '/src/utils/services/getCoordsFromAddress';
-
 import styles from './GoogleMapComponent.module.scss';
+import CSS from 'csstype';
 
-export default function GoogleMapComp({ houses, style }) {
-  function Map() {
-    const [housesCoords, setHousesCoords] = useState([]);
 
+export default function GoogleMapComp({ houses, style }:{houses:House,style:CSS.Properties}) {
+  function Map() {
+    let [housesCoords, setHousesCoords]:[Coord[],Dispatch<SetStateAction<Coord[]>>] = useState<Coord[]>([]);
```

Comment:
You don't have to declare the types of `[state, setState]` right after declaring them as TS will know their types thanks to the way it works with React, so
`let [housesCoords, setHousesCoords] = useState<Coord[]>([]);`
will do and using `const` instead `let` would be advisable. Sometimes you don't have to declare the type even inside the <> after `useState` but it's generally a good practise to do so even when the type of a state can be guessed from an initial value of a state.